docs/conversion/sakai_20_mysql_conversion.sql

```diff
@@ -13,6 +13,12 @@ ALTER TABLE POLL_OPTION ADD OPTION_ORDER INTEGER;
 
 -- END SAK-41391
 
+-- SAK-47205
+-- This will target orphaned poll_option rows that the Backfill poll option order quartz job may miss.
+-- The Backfill poll option order job should be run after applying this conversion
+UPDATE poll_option set option_order = 0 where option_id in (select * from (select a.option_id from poll_option as a left outer join poll_poll as b on a.option_poll_id = b.poll_id where b.poll_id is null) as t);
```

Comment:
select * from seems redundant?

Reply:
I originally tried without it but MySQL wouldn't let me update the table that it was selecting from.  The extra select *  works around that.  Please let me know if there's a better/safer way to accomplish this.

Reply:
So you're trying to set option_order=0 where the poll does not exist in poll_poll?

Shouldn't these be deleted instead?

Reply:
That's certainly another option. Pun intended.